source/guides/stacks/mers.rst

```diff
@@ -104,58 +148,58 @@ Components of the |MERS| include:
    licensing and usage of the Media Reference Stack.
 
 
-Getting the pre-built |MERS| container image
-********************************************
+Get the pre-built |MERS| container image
+****************************************
 
-Pre-built |MERS| Docker images are available on DockerHub at
-https://hub.docker.com/r/clearlinux/stacks-mers
+Pre-built |MERS| Docker images are available on DockerHub* at
+https://hub.docker.com/r/sysstacks/mers-clearlinux
 
 
 To use the |MERS|:
 
 #. Pull the image directly from `Docker Hub
-   <https://hub.docker.com/r/clearlinux/stacks-mers>`_. 
+   <https://hub.docker.com/r/sysstacks/mers-clearlinux>`_. 
 
    .. code-block:: bash
 
-      docker pull clearlinux/stacks-mers
+      docker pull sysstacks/mers-clearlinux
 
    .. note ::
 
       The |MERS| docker image is large in size and will take some time to
       download depending on your Internet connection.
 
       If you are on a network with outbound proxies, be sure to configure
-      Docker allow access. See the `Docker service proxy
+      Docker to allow access. See the `Docker service proxy
       <https://docs.docker.com/config/daemon/systemd/#httphttps-proxy>`_ and
       `Docker client proxy
       <https://docs.docker.com/network/proxy/#configure-the-docker-client>`_
       documentation for more details.
       
-#. Once you have downloaded the image, run it with:
+#. Once you have downloaded the image, run it using the following command:
 
    .. code-block:: bash
 
-      docker run -it clearlinux/stacks-mers
+      docker run -it sysstacks/mers-clearlinux
```

Comment:
@puneetse please include the parameters ` --device=/dev/dri -e QSV_DEVICE=/dev/dri/renderD128 ` before the image name

Reply:
The `docker run` with those parameters explained further down in the document in the examples. The idea is to start simple and build on complexity.